Update changes for NVMe Drives (#1454)
astronomer/docs: astro/release-notes.md

### Write to temporary storage on AWS clusters

> AWS clusters that use `m5d` and `m6id` worker types can now use the full capacity of the NVMe SSD volumes available in these instances. Prior to this release, Astro used the utilization of the root volume to make scheduling decisions; with this change  the entire storage capacity is examined. 

Reviewer: ```suggestion
AWS clusters that use `m5d` and `m6id` worker types now use the full capacity of the available NVMe SSD volumes. Previously, Astro used only the root volume when making scheduling decisions. 
```

Reviewer 2: @kwcanuck @dnarain-astro Can we condense this change to be a bug fix? The additional docs for how this storage is configured is already set to publish in https://github.com/astronomer/docs/pull/1404/

Reviewer: @jwitz makes sense to me.

> Tasks using KubernetesPodOperator can create  mounts to any path, but other tasks must write to `/ephemeral` to use this storage.  

Reviewer: ```suggestion
Tasks using KubernetesPodOperator can create mounts to any path, but all other tasks must write to `/ephemeral` .  
```

> **Note that non-KPO tasks that used local storage on these instances will need to be adjusted to write to `/ephemeral` instead.**

Reviewer: ```suggestion
**Non-KPO tasks that use local storage will need to be updated to write to `/ephemeral`.**
```

> You can use this storage for simple operations such as a disk-based merge sort or checkpointing to prevent crashes. To use these worker types on your cluster, see [Modify a cluster](modify-cluster.md) and [Configure worker queues](configure-worker-queues.md). 

Reviewer: ```suggestion
You can use temporary storage for simple operations such as a disk-based merge sort or checkpointing. To use these worker types on your cluster, see [Modify a cluster](modify-cluster.md) and [Configure worker queues](configure-worker-queues.md). 
```
